tmpl/jenkins/Jenkinsfile.j2

```diff
@@ -32,7 +32,6 @@ pipeline {
             steps {
                 sh '''#!/usr/bin/env bash
                 set -e
-                cd ${WORKSPACE}/{{ project_slug }}
```

Comment:
we could leave the `cd ${WORKSPACE}` in if we want to be explicit about the directory we're in - but it should not be necessary

Reply:
yes, I always thinking about it before removing. but then I thought that the assumption is that at the beginning of each stage we are always in the WORKSPACE of the jenkins plan, and so it was unnecessary